### ISSUE TITLE
[Unit Test]: Test for correct job landmarks

### ISSUE THREAD

**Issue by PowerfulBacon** [BeeStation/BeeStation-Hornet#11407]
Test that maps have all job landmarks placed. (All jobs that haven't been disabled on the map)

**Comment by EvilDragonfiend** [comment]
no, we don't need this, or should be flagged for each map. some jobs like viro wouldn't work.

**Comment by PowerfulBacon** [comment]
That's how the new unit tests work, they are run per-map
